templates/openstack/architecture.html

```diff
@@ -0,0 +1,280 @@
+        </div>
+        <div class="p-equal-height-row__col">
+          <div class="p-equal-height-row__item">
+            <h3 class="p-heading--5">Full bottom-up automation</h3>
+          </div>
+          <p class="p-equal-height-row__item">
+            Canonical OpenStack uses full bottom-up automation based on <a href="https://maas.io/">MAAS</a> and <a href="https://juju.is/">Juju</a> across all the layers of the stack. This covers the initial provisioning of physical machines, cloud bootstrap processes as well as its post-deployment operations.
+          </p>
+        </div>
+        <div class="p-equal-height-row__col">
+          <div class="p-equal-height-row__item">
+            <h3 class="p-heading--5">High-level abstraction</h3>
+          </div>
+          <p class="p-equal-height-row__item">
+            Canonical OpenStack exposes a high-level interface for bootstrapping and operating the cloud. This effectively abstracts the complexity of the upstream project and enables users to spin up a cloud in just a few simple steps.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>OpenStack under the hood</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container--cinematic is-highlighted u-hide--medium u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/ccb38017-openstack-under-the-hood.png",
+                        alt="",
+                        width="1200",
+                        height="501",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="p-section--shallow">
+          <p>
+            OpenStack uses a modular architecture with every service responsible for an individual cloud feature. This includes identity management, an image catalogue as well as all the compute, network and storage functions that are typically required to provision and decommission virtual machines.
+          </p>
+          <p>
+            The services are further broken down into processes based on a microservices architecture. Each process implements an atomic cloud function, such as handling user requests, proxying database queries or invoking various types of backend technologies used by OpenStack underneath.
+          </p>
+          <p>
+            In order to minimise its footprint, Canonical OpenStack ships with only the key features enabled by default. Additional features, such as CaaS or a Load Balancer can be enabled post-deployment with a single terminal command.
+          </p>
+        </div>
+        <hr class="p-rule--muted" />
+        <a class="p-button--positive" href="https://microstack.run/docs">Access supported features</a>
```

Comment:
All microstack links will have to change now when we have shifted it to 
https://canonical.com/microstack

Reply:
There will be a separate request from the stakeholder